src/android/com/plugin/gcm/OneSignalPush.java

```diff
@@ -241,6 +242,12 @@ public void onOSSubscriptionChanged(OSSubscriptionStateChanges stateChanges) {
       }
       result = true;
     }
+    else if (REMOVE_SUBSCRIPTION_OBSERVER.equals(action)) {
+      if (subscriptionObserver == null) {
```

Comment:
Shouldn't be != and not == ?

Reply:
correct. Thanks for the repair.

Pushed the new changes, both for android and ios.